core/deployment/src/main/java/io/quarkus/deployment/builditem/nativeimage/AutoFeatureAugmentor.java

```diff
@@ -0,0 +1,7 @@
+package io.quarkus.deployment.builditem.nativeimage;
+
+import io.quarkus.gizmo.BytecodeCreator;
+
+public interface AutoFeatureAugmentor {
+    void augment(BytecodeCreator overallCatch);
```

Comment:
I don't think we should be doing this. There is no limit on the number of features you can have to just create a new one with `@AutomaticFeature` (no need to even generate it, although you can if you need to).